Classes/Controller/FormController.php

```diff
@@ -463,17 +460,9 @@ public function formAction(): ResponseInterface {
     $this->prepareFormSets();
 
     if ('json' == $this->formConfig->responseType) {
-      $this->jsonResponse->steps = $this->formConfig->steps;
```

Comment:
I'm note sure if this is important. but this is missing now.

Reply:
I moved that to `prepareJsonResponse()`. It would allways be one submit back here anyways

Reply:
au contraire mon ami...

You moved 

`$this->jsonResponse->step = $this->formConfig->step;` 

but this  is

`$this->jsonResponse->steps = $this->formConfig->steps;`

see the s at the end? Even small details matter. ;) 

Thats why I said I don't know if steps is important, but I guess it is not not important.